python/paddle/trainer/config_parser.py

```diff
@@ -1571,7 +1571,13 @@ def __init__(self, name, inputs, softmax_selfnorm_alpha=0.1, **xargs):
 
 @config_layer('fc')
 class FCLayer(LayerBase):
-    def __init__(self, name, size, inputs, bias=True, **xargs):
+    def __init__(self,
+                 name,
+                 size,
+                 inputs,
+                 bias=True,
+                 error_clipping_threshold=None,
```

Comment:
It's better to put `error_clipping_threshold ` in the base layer, since the `error_clipping_threshold` can be used in many layers.